.circleci/config.yml

```
@@ -265,13 +267,39 @@ jobs:
             bin/smoke_test --remote --no-source-env
       - notify-slack-smoke-test-status
       - store-smoketest-results
+  check-pinpoint-config:
+    docker:
+      - image: circleci/ruby:2.6-node-browsers
+    steps:
+      - checkout
+      - bundle-yarn-install
+      - run:
+          name: Check current AWS Pinpoint country support
+          command: |-
+            diff <(bundle exec ./scripts/pinpoint-supported-countries) config/country_dialing_codes.yml
```

this will fail right now so we need `|| true ` so it doesn't spam us

```suggestion
            diff <(bundle exec ./scripts/pinpoint-supported-countries) config/country_dialing_codes.yml || true # manually disabled countries due to spam
```

---

Whoops nevermind I saw the 1964223 changes, same same

---

> this will fail right now so we need `|| true ` so it doesn't spam us

The expectation is that `ignore: main` would prevent this whole task from being run at all.

---

> Whoops nevermind I saw the [1964223](https://github.com/18F/identity-idp/commit/196422363e21bc995027c0795c77c1d09845d92e) changes, same same

And whoops, I just saw your follow-up comment 😄